### ISSUE TITLE
Réflexion sur l'utilisation de GitHub pour le HW

### ISSUE THREAD

**Issue by martintimothee** [IPM-France/sandbox#31]
Après la prez de Fred, je me dis que pour le hard nous ne sommes pas pret à utiliser GitHub comme d'un gestionneur de version, notre interface obligatoire avec Baan reduisant l'interet...
Par contre c'est un outil de communication.
Mais l'interet que j'y vois à breve écheance c'est de l'utiliser comme d'un gestionnaire des taches...
Qu'en pensez-vous?

**Comment by beaugiraudv** [comment]
Ok avec cette analyse. L'autre utilisation sera d'extraire les charges de ces taches "ready" et "in progress" pour les injecter dans le GAANT que nous revoyons de manière hebdomadaire (planning, priorités, ...).
@martintimothee @jveyrie 
Avez vous un avis à ce stade sur les pistes proposées par Fred : est ce que l'on reste sur un seul Repository (= Hardware) et on suit nos composants hw (EK1000, 2000, ...) par des labels ? Vu l'utilisation ci-dessus décrite, cela me semble bien suffisant, pour moi inutile de créer autant de projets que de modèles de borne.
@mohierf : sur la piste évoquée hier pour les RT d'ajouter un label (ou milestone) numéro de projet BAAN (ex de la borne pour la TAM : 4615) sur l'ensemble des taches, et si intérêt confirmé (facilité, traçabilité ISO, ...), est ce que Waffle saurait donner une vue de toutes les taches taguées 4675 sur plusieurs projets (Hardware, Borne, ...) ?

**Comment by mohierf** [comment]
Waffle permet sans problème de voir toutes les issues qui portent le même label. Donc, par exemple, on pourra voir toutes les issues ayant le tag `4615-TAM` ...

**Comment by jveyrie** [comment]
Effectivement, un seul repository Hardware parait suffisant dans un premier temps. Par contre je ne vois pas bien comment l'organiser ensuite ? Si on compare au sandbox actuel, je vois pas à quoi correspondent les lignes qui s'affichent en première page (des sous répertoires, des taches, des ??). Sachant que pour l'instant on créé des issues que l'on peut consulter seulement en cliquant sur le coté ... bref j'ai besoin encore d'entrainement ;-)

**Comment by beaugiraudv** [comment]
Github intègre une notion de Filters, mais le choix me semble en effet trop limité (sinon synthaxe recherche ?).
Par contre l'interface Waffle semble simple et suffisante pour notre usage : après choix du board `ipm-france/sandbox`, le menu `Flter board` en haut à droite permet de filtrer les issues only.
Filtre sur les ressources assignées, les milestones et les labels.
De ce que je constate, c'est un **ET logique** qui s'applique, permettant par exemple de voir toutes les issues affectées @jveyrie , contenant les labels ek2015 ET enhancement ET ...

**Comment by beaugiraudv** [comment]
Pour trace ci-joint la liste des labels discutée dans nos échanges.
A discuter demain
[Labels Hardware.docx](https://github.com/IPM-France/sandbox/files/39286/Labels.Hardware.docx)
